src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -371,24 +438,27 @@ def minimize_corpus(self, target_path, arguments, input_dirs, output_dir,
     """
     runner = libfuzzer.get_runner(target_path)
     libfuzzer.set_sanitizer_options(target_path)
-    merge_tmp_dir = self._create_temp_corpus_dir('merge-workdir')
+    if not self.merge_tmp_dir:
+      self.merge_tmp_dir = self._create_temp_corpus_dir('merge-workdir')
 
-    merge_result = runner.merge(
+    result = runner.merge(
         [output_dir] + input_dirs,
         merge_timeout=max_time,
-        tmp_dir=merge_tmp_dir,
+        tmp_dir=self.merge_tmp_dir,
         additional_args=arguments,
         artifact_prefix=reproducers_dir)
 
-    if merge_result.timed_out:
+    if result.timed_out:
       raise MergeError('Merging new testcases timed out')
 
-    if merge_result.return_code != 0:
+    if result.return_code != 0:
       raise MergeError('Merging new testcases failed')
 
+    merge_stats = stats.parse_stats_from_merge_log(result.output.splitlines())
```

Comment:
One more thing I've noticed while staging: when merge times out, we shouldn't have any incremental stats. I think it's coming from the old code, maybe I should delete it right in this CL -- will think about it.

Reply:
I've deleted edge and feature coverage stats parsing from the fuzzing log. Now all coverage stats we have will be coming from the merge step. If numbers are zero, that means merge for a fuzz target fails to finish.

Filed https://github.com/google/clusterfuzz/issues/1254 to see how often fuzz targets fail to merge. Maybe there are issues we should fix, maybe things are alright, will see.